platform-modules/batch-models/src/main/scala/org/ekstep/analytics/util/AssessmentReportUtil.scala

```diff
@@ -0,0 +1,158 @@
+package org.ekstep.analytics.util
+
+import java.io.File
+import java.nio.file.{Files, StandardCopyOption}
+
+import org.apache.spark.sql.functions.col
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.ekstep.analytics.framework.Level.{ERROR, INFO}
+import org.ekstep.analytics.framework.util.{JSONUtils, JobLogger}
+import org.elasticsearch.spark.sql._
+import org.joda.time.DateTime
+import org.joda.time.format.DateTimeFormat
+import org.sunbird.cloud.storage.conf.AppConf
+import org.sunbird.cloud.storage.factory.{StorageConfig, StorageServiceFactory}
+
+object AssessmentReportUtil {
```

Comment:
Move file handling logic to File Util and http logic to http utils

Reply:
Created file util to handle the file operations